app/controllers/api/v1/collaterals_controller.rb

```diff
@@ -1,36 +1,36 @@
 module Api
   module V1
     class CollateralsController < ApplicationController
-      skip_before_action :verify_authenticity_token
-
       def index
         render json: Collaterals::IndexService.call
       end
 
       def show
-        render json: collateral
+        collateral_id = params[:id]
+
+        render json: Collaterals::ShowService.call(collateral_id)
       end
 
       def create
-        render json: Collaterals::CreateService.call(collateral_params)
+        data = params.require(:data).permit(attributes: [:name, :kind_id, :url])['attributes']
+        dto = Collaterals::CreateCollateralDto.new(data)
+
+        render json: Collaterals::CreateService.call(dto)
```

Comment:
RFC:
is `data = params.require(:data).permit(attributes: [:name, :kind_id, :url])['attributes']` necessary when we have DTO? 
If DTO "controls" the params passed to service, then `params.require().permit()` seems to be duplicating the same work. 
What do you think? 
```
dto = Collaterals::CreateCollateralDto.new(params[:data][:attributes])
```
will be sufficient?

Reply:
I thinkk about it as basic validator - we require specific fields and cut object to them. We can discuss this matter in Monday. I think changing that won't take much time if we decide to remove this or add more strict validation.